code/game/objects/items/paintkit.dm

```diff
@@ -149,6 +149,52 @@
 		return
 	return ..()
 
+/obj/item/device/kit/suit/rig
+	name = "rig modification kit"
+	desc = "A kit for modifying a rigsuit."
+	uses = 1
+
+/obj/item/device/kit/suit/rig/customize(var/obj/item/I, var/mob/user)

```

Comment:
Pointers set here in customize. Only the RIG uses the kit's icon, because they're horrifying and use all the same icon_state. Changing this would mean touching RIGs way more than I'd like, because it's already a streamlined for loop to assign everything the icons/states in multiple places and on creation.